src/HotChocolate/Language/src/Language.SyntaxTree/FieldNode.cs

```diff
@@ -190,4 +191,44 @@ public FieldNode WithSelectionSet(SelectionSetNode? selectionSet)
     /// </returns>
     public FieldNode WithRequired(INullabilityNode? required)
         => new(Location, Name, Alias, required, Directives, Arguments, SelectionSet);
+
+    public bool Equals(FieldNode? other)
```

Comment:
can you copy the xml comments for equals. They are in one of the example classes I made.